### PR TITLE
Only set mimetype if not already set.

### LINKED ISSUE #45: non-API views attached to flump blueprint get jsonapi content type
It seems that any endpoint attached to a flump blueprint will get given the jsonapi content-type.  Sometimes it's desirable to put non-api views on flump blueprints, so it'd be good if we could fix this somehow...

### PR DESCRIPTION
**Pull request by obmarg**
This should allow us to support API endpoints that don't want to have a
jsonapi mimetype.  For example, any user facing endpoints that present
HTML will not want to present the jsonapi mimetype.

Fixes #45

### CODE REVIEW

**Comment by carlos-alberto** [comment]
:shipit:

**Comment by lukasz-madon** [comment]
LGTM

**Comment by obmarg** [comment]
Not entirely sure if this is enough...

**Comment by carlos-alberto** [comment]
Add a test?

**Comment by obmarg** [comment]
🔁

File: flump/view.py [hunk 1]
```diff
@@ -153,6 +154,13 @@ def _build_entity_data(self, entity):
                           entity, EntityMetaData(self._get_etag(entity)))
 
 
+def add_content_type(response):
```
Comment (carlos-alberto) [inline flump/view.py]: Don't you only want to do this if it isn't already there?

Comment (obmarg) [inline flump/view.py]: In this case I think it's safe to just do it blanket: if we've got a flump view then we know it's jsonapi...

Comment (carlos-alberto) [inline flump/view.py]: Ah good point. :+1:

**Comment by carlos-alberto** [comment]
:shipit:

**Pull request merged**